src/components/elements/off-canvas-menu.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useRef, useEffect } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { LocalizedLink } from '../localization'
+import Arrow from 'images/svg/arrow.svg'
+
+const OffCanvasMenu = styled.section`
+    position: fixed;
+    background-color: var(--color-white);
+    top: 0;
+    height: 100vh;
+    width: 238px;
+    opacity: 0.98;
+    transition: right 0.4s;
+    right: ${props => (props.is_canvas_menu_open ? '0' : '-238px')};
```

Comment:
can we use rem here?

Reply:
Yes we can but since we have this off-canvas menu just in mobile, I prefer to use px instead of rem. Also, if we use rem it will be bigger in larger screen size devices, but we don't want to.
@oskar-binary what is your opinion?

Reply:
@arash-fs For me a fixed size for the mobile menu is fine here (px)